frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -381,13 +382,6 @@ const ManageQueriesPage = ({
     isTeamMaintainer ||
     isObserverPlus; // isObserverPlus checks global and selected team
 
-  const hideQueryActions =
-    // there are no filters and no returned queries, indicating there are no global/team queries at all
-    !(!!location.query.query || !!location.query.platform) &&
-    !queriesResponse?.count &&
-    // the user has permission
-    (!isOnlyObserver || isObserverPlus || isAnyTeamObserverPlus);
-
```

Comment:
This was used to hide the top-right "Add Query" button; we always want to show this now as long as the user is permitted to add queries.